### ISSUE TITLE
normalizeDiacritics doesn't preserve ß (Eszett)

### ISSUE THREAD

**Issue by chocolateboy** [VitorLuizC/normalize-text#9]
**Node**: v14.15.4 | **OS**: Linux (Arch) | **normalize-text**: v2.3.1

`normalizeDiacritics` unexpectedly removes the [letter ß](https://en.wikipedia.org/wiki/%C3%9F):

<table>
<tr><th align="right">Input</th><td>"Amélie plays Fußball"</td></tr>
<tr><th align="right">Expected</th><td>"Amelie plays Fußball" or "Amelie plays Fussball"</td></tr>
<tr><th align="right">Actual</th><td>"Amelie plays Fuball"</td></tr>
</table>

**Comment by chocolateboy** [comment]
Looks like the substitution pattern is too wide and is removing non-diacritics:

https://github.com/VitorLuizC/normalize-text/blob/d82bc15d75303dead4831d1a7f2c0d19c7ed742d/src/normalizeDiacritics.ts#L12

The character class listed [here](https://stackoverflow.com/a/37511463) ([via](https://github.com/vuejs/vuepress/pull/1855#issuecomment-536289974)) appears to work as expected:

```javascript
const normalize = str => str.normalize('NFKD').replace(/[\u0300-\u036F]/g, '')

normalize('Amélie plays Fußball') // "Amelie plays Fußball"
```

**Comment by VitorLuizC** [comment]
Fixed and released under version [`2.3.2`](https://github.com/VitorLuizC/normalize-text/releases/tag/2.3.2).
